### PR TITLE
#147 remove RSpecCommand API

### PR DESCRIPTION
**Pull request by CAMOBAP**

### CODE REVIEW

File: spec/metanorma_spec.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 RSpec.describe Metanorma do
   it "processes metanorma options inside Asciidoc" do
-    File.open("test.adoc", "w:UTF-8") { |f| f.write(ASCIIDOC_PREAMBLE_HDR) }
+    File.write("test.adoc", ASCIIDOC_PREAMBLE_HDR, encoding: "UTF-8")
```
Comment (abunashir) [inline spec/metanorma_spec.rb:5]: @CAMOBAP: Looks like we are doing this write and cleanup in each test, what do you think about extracting those to a separate method or something? And also maybe use a `temp` directory, so it doesn't clutter when we need run those test locally?

File: spec/metanorma_spec.rb [hunk 2]
```diff
@@ -140,32 +140,34 @@ def ruby(x)
...
-end
 
-RSpec.describe "warns when no standard type provided" do
-  file "test.adoc", ASCIIDOC_CONFIGURED_HDR
-  command "metanorma test.adoc"
-  its(:stdout) { is_expected.to include "Please specify a standard type" }
-end
+  it "warns when no standard type provided" do
+    File.write("test.adoc", ASCIIDOC_CONFIGURED_HDR, encoding: "UTF-8")
+    stdout = `metanorma test.adoc`
```
Comment (abunashir) [inline spec/metanorma_spec.rb:146]: Good idea, if you need more control over the `stdout` or `stderr` then I think we also have test helpers here - https://github.com/metanorma/metanorma-cli/blob/master/spec/support/console_helper.rb

Comment (CAMOBAP) [reply spec/metanorma_spec.rb:146]: `` or `Open3`, looks much cleaner and native, what benefits of `capture_stdout`/`capture_stderr` against them?

File: spec/metanorma_spec.rb (hunk 1 above)
Comment (ronaldtse) [reply spec/metanorma_spec.rb:5]: Agree with @abunashir .

File: spec/metanorma_spec.rb (hunk 2 above)
Comment (abunashir) [reply spec/metanorma_spec.rb:146]: I'm not exactly sure anymore why we wrote our own version, but I think there was some issue with RSpec or something back then. If there is no incompatibility with `open3` and our test suite then please feel free to migrate it 👍

File: spec/metanorma_spec.rb (hunk 1 above)
Comment (CAMOBAP) [reply spec/metanorma_spec.rb:16]: Updated (in separate commit), @ronaldtse @abunashir could you please have a look https://github.com/metanorma/metanorma-cli/pull/148/commits/e04ddef3ffb363a41858da8ba4e22c977d93b94a

**Comment by ronaldtse** [comment]
Agree with @abunashir. Can we get that comment implemented and then merge? Thanks @CAMOBAP !

**Comment by CAMOBAP** [comment]
@ronaldtse @abunashir done

**Comment by ronaldtse** [comment]
Looks great, thanks!

**Pull request merged**